bsp/stm32f10x-HAL/drivers/drv_sdcard.c

```diff
@@ -0,0 +1,330 @@
+/*
+ * File      : drv_sdcard.c
+ * This file is part of RT-Thread RTOS
+ * COPYRIGHT (C) 2018, RT-Thread Development Team
+ *
+ * The license and distribution terms for this file may be
+ * found in the file LICENSE in this distribution or at
+ * http://www.rt-thread.org/license/LICENSE
```

Comment:
Please make sure the file header is the latest. You can refer to here:
https://github.com/RT-Thread/rtthread-specification/blob/master/bsp_skeleton/applications/main.c